package.json

```diff
@@ -37,17 +37,18 @@
     "fs-extra": "^7.0.1",
     "ignore-walk": "^3.0.2",
     "inflection": "^1.12.0",
-    "ts-jest": "^24.1.0",
     "yargs-parser": "^13.0.0"
   },
   "devDependencies": {
     "@types/jest": "^24.0.19",
+    "@types/node": "^13.7.0",
```

Comment:
should fix https://ci.appveyor.com/project/jondot/hygen/builds/30699794/job/k9xt32wlefgev6vn

Reply:
moving to a new pr